Content.Server/Destructible/Thresholds/Behaviors/BurnBodyBehavior.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Content.Shared.Body.Components;
+using Content.Shared.Inventory;
+using Content.Shared.Popups;
+using JetBrains.Annotations;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Destructible.Thresholds.Behaviors
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class BurnBodyBehavior : IThresholdBehavior
+    {
+
+        public void Execute(EntityUid bodyId, DestructibleSystem system, EntityUid? cause = null)
+        {
+            var transformSystem = system.EntityManager.System<TransformSystem>();
+            var inventorySystem = system.EntityManager.System<InventorySystem>();
+            var sharedPopupSystem = system.EntityManager.System<SharedPopupSystem>();
+
+            if (!system.EntityManager.TryGetComponent<InventoryComponent>(bodyId, out var comp))
+                return;
+
+            foreach (var item in inventorySystem.GetHandOrInventoryEntities(bodyId))
+            {
+                transformSystem.AttachToGridOrMap(item);
+            }
+
```

Comment:
Something to consider if someone is cooking inside a locked container this will dump their stuff outside of it instead.

Reply:
gonna fix this in itemspawn behavior as well, it has the same issue